public/test/matchers/types.ts

```diff
@@ -0,0 +1,13 @@
+import { Observable } from 'rxjs';
+
+export const OBSERVABLE_TEST_TIMEOUT_IN_MS = 1000;
```

Comment:
I wonder if it would make sense to tie it to jest `testTimeout` config value? I am not sure how to get it easily though, don't see any access for that in jest object :(. Defaults can be accessed with `const {defaults} = require('jest-config');` but you would need to reread the configs to get current value which is not great.

Reply:
The jest default is 5000ms, that is too long, so I skipped it. But good suggestion.